ansible/playbooks/adhoc/uninstall.yml

```diff
@@ -33,21 +47,46 @@
         - kube-scheduler
         - kube-addons
         - etcd
+        - etcd2
       failed_when: false
 
     - name: Remove packages
       action: "{{ ansible_pkg_mgr }} name={{ item }} state=absent"
-      when: not is_atomic | bool
+      when: not is_coreos and not is_atomic | bool
       with_items:
         - flannel
         - kubernetes-node
         - kubernetes-master
         - etcd
+        - etcd2
         - kube-addons
         - atomic-enterprise
         - atomic-enterprise-master
         - atomic-enterprise-node
 
+    - name: CoreOS | Remove packages
```

Comment:
small nit ^ to change the name from packages to bins and systemd files. I just want to make sure we don;t confuse anyone. This change can be combined with the other one in the same future PR.